main/src/com/google/refine/RefineServlet.java

```diff
@@ -73,8 +73,7 @@ public class RefineServlet extends Butterfly {
     static public String FULLNAME = "OpenRefine ";
 
     static final long serialVersionUID = 2386057901503517403L;
-
-    static private final String JAVAX_SERVLET_CONTEXT_TEMPDIR = "javax.servlet.context.tempdir";
+    static private final String JAKARTA_SERVLET_CONTEXT_TEMPDIR = "jakarta.servlet.context.tempdir";
```

Comment:
I assume this just boils up to the systems tempdir? or is it a special separate folder that we might need to have a preference for?